.goreleaser.yml

```diff
@@ -72,7 +72,7 @@ brews:
     - name: git
       type: optional
     install: |-
-      bin.install "symfony"
+      bin.install "symfony-cli"
```

Comment:
Would probably be better to keep `symfony` as the binary name instead.

Reply:
I agree,

Reply:
I think this is the correct way to solve it: https://github.com/symfony-cli/symfony-cli/pull/3